.github/workflows/release.yml

```diff
@@ -50,7 +60,8 @@ jobs:
         with:
           config: config.yaml
         env:
-          CR_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          CR_KEY: "${{ secrets.GPG_KEY_ID }}"
```

Comment:
This env variable is not used by any tools behind, afaik.
Is there a reason for it?

Reply:
Huh, the example I found implied you could use it to specify which GPG key was used for signing, but I just checked out the chart-releaser code and there's no mention of it so maybe it's old or just invalid. I was doing it for belt-and-suspenders reasons in case we end up needing to have more than one key handy in the secrets, but it should be unnecessary when there's only one.